src/qt/pivx/qtutils.cpp

```diff
@@ -122,6 +122,36 @@ QPixmap encodeToQr(QString str, QString &errorStr, QColor qrColor){
     return QPixmap();
 }
 
+void setFilterAddressBook(QComboBox* filter, SortEdit* lineEdit) {
+    initComboBox(filter, lineEdit);
+    filter->addItem("Receiving", 0);
+    filter->addItem("Contacts", 1);
```

Comment:
Feat request: would be nice to have also Delegators and Cold-Staking send/receive addresses (`AddressTableModel::Delegators`, `AddressTableModel::ColdStaking`, and `AddressTableModel::ColdStakingSend`).

And perhaps also an option "All" to export everything (would make it the default).

Reply:
Sounds good. Adding the functionality and a new column "type" in the exported file too.